charts/pvc-autoresizer/templates/controller/deployment.yaml

```
@@ -14,6 +14,10 @@ spec:
     metadata:
       labels:
         app.kubernetes.io/name: pvc-autoresizer
+      {{ if .Values.controller.podAnnotations -}}
+      annotations:
+        {{- toYaml .Values.controller.podAnnotations | nindent 8 }}
+      {{ end -}}
```

```suggestion
      {{- end }}
```

The right trim marker changes the indent of `spec` in the following line. We must keep it.

```diff
@@ -183,7 +183,7 @@
     metadata:
       labels:
         app.kubernetes.io/name: pvc-autoresizer
-    spec:
+      spec:
       serviceAccountName: release-name-pvc-autoresizer-controller
       containers:
         - name: pvc-autoresizer
```